doc/ref/gappkg.xml

```diff
@@ -173,5 +184,2074 @@ users.
+<P/>
+Nowadays, facilities to distribute a document over several files
+to allow the documentation for parts of some code to be stored in
+the same file as the code itself are provided by the &GAPDoc; package (see
+<Ref Sect="Distributing a Document into Several Files" BookName="gapdoc"/>).
+The same approach is demonstrated by the <Package>Example</Package> package.
+E.g. <F>example/doc/example.xml</F> has the statement 
+<C>&lt;#Include Label="ListDirectory"></C>
+and <F>example/lib/files.gd</F> contains
+<Log><![CDATA[
+#############################################################################
+##
+#F  ListDirectory([<dir>])  . . . . . . . . . . list the files in a directory
+##
+##  <#GAPDoc Label="ListDirectory">
+##  <ManSection>
+##  <Func Name="ListDirectory" Arg="[dir]"/>
+##
+##  <Description>
+##  lists the files in directory <A>dir</A> (a string) 
+##  or the current directory if called with no arguments.
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+DeclareGlobalFunction( "ListDirectory" );
+]]></Log>
+This is all put together in the file <F>example/makedocrel.g</F> which 
+builds the package documentation, calling 
+<Ref Func="MakeGAPDocDoc" BookName="gapdoc"/> with 
+locations of library files containing parts of the 
+documentation.
+
+</Section>
+
 
+
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="Creating the PackageInfo.g File">
+<Heading>Creating the PackageInfo.g File</Heading>
+<Index Key="ValidatePackageInfo"><C>ValidatePackageInfo</C></Index>
+As a first step the example in <Ref Sect="An Example of a GAP Package"/>
+shows the information needed for the package loading mechanism to load 
+a simple package. However, if your  package  depends on  the  functionality  
+of other packages, the component <C>Dependencies</C>  given in  the
+<F>PackageInfo.g</F> file becomes  important
+(see <Ref Label="Package dependencies"/> below),
+and more entries become relevant if you want to 
+distribute  your package: they  contain lists  of authors and/or  
+maintainers including contact information,  URLs of the  package archives 
+and  README files, status information, text for a package overview webpage,
+and so on.
+<P/> 
+We suggest to create a <F>PackageInfo.g</F> file for your package by
+copying the one in the <Package>Example</Package> package, distributed
+with &GAP;, or using the <Package>PackageMaker</Package>
+(<URL>https://github.com/gap-system/PackageMaker</URL>), and then adjusting
+it for your package. Within &GAP; you can look at this template file for a
+list and explanation of all recognized entries by
+<Log><![CDATA[
+Pager(StringFile(Filename(DirectoriesLibrary(), 
+                          "../pkg/example/PackageInfo.g")));
+]]></Log>
+<P/>
+Once you have created the <F>PackageInfo.g</F> file for your package,
+you can test its validity with the function 
+<Ref Func="ValidatePackageInfo"/>.
+
+</Section>
+
```

Comment:
I stopped reading here for now. Perhaps somebody else can pick up reviewing after this, or perhaps I'll find some time and energy to do it later.